pkg/virt-controller/watch/migration/migration.go

```diff
@@ -574,6 +587,20 @@ func (c *Controller) updateStatus(migration *virtv1.VirtualMachineInstanceMigrat
 	return nil
 }
 
+func (c *Controller) setSynchronizationAddressStatus(migration *virtv1.VirtualMachineInstanceMigration) error {
+	kvs := c.kubevirtStore.List()
+	if len(kvs) > 1 {
+		log.Log.Errorf("More than one KubeVirt custom resource detected: %v", len(kvs))
+		return fmt.Errorf("more than one KubeVirt custom resource detected: %v", len(kvs))
+	}
+
+	if len(kvs) == 1 {
+		kv := kvs[0].(*virtv1.KubeVirt)
```

Comment:
Sometime you check type assertions, sometimes you don't. Does it make sense to add a check here too?

Reply:
I like checking the type assertion, must have been an oversight. Let me fix that. I know other people have other opinions, but I rather be safe.